roles/kubernetes/preinstall/tasks/0020-verify-settings.yml

```diff
@@ -282,3 +282,19 @@
     that: ansible_os_family not in ["Flatcar Container Linux by Kinvolk"]
     msg: "download_run_once not supported for Flatcar Container Linux"
   when: download_run_once or download_force_cache
+
+- name: Verify synchronized time before install/upgrade
+  shell: "set -o pipefail && timedatectl status | grep synchronized | awk '{ print $4 }'"
+  args:
+    executable: /bin/bash
+  register: result
+  run_once: yes
```

Comment:
why run_once ?

should check time synchronize on all host

Reply:
on centos 7 command `timedatectl status | grep synchronized | awk '{ print $4 }'` Return empty string

yes in 3 position
``` timedatectl status
      Local time: Tue 2020-10-06 12:38:57 UTC
  Universal time: Tue 2020-10-06 12:38:57 UTC
        RTC time: Tue 2020-10-06 12:38:56
       Time zone: UTC (UTC, +0000)
     NTP enabled: yes
NTP synchronized: yes
 RTC in local TZ: no
      DST active: n/a
```

Reply:
Thank you. and let me remove `run_once`.